.github/github-ci.yaml

```
@@ -0,0 +1,34 @@
+        with:
+          node-version: 20
+          cache: 'pnpm'
+
+      - name: Install pnpm
+        run: npm install -g pnpm
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Build project
+        run: pnpm run build
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)